<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Black_jack</class>
 <widget class="QWidget" name="Black_jack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="karta_igraca_1">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>310</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="spil">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>40</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sledeca_karta">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>300</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Sledeca karta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ne_zelim">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>330</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Ne zelim da vucem</string>
   </property>
  </widget>
  <widget class="QPushButton" name="podeli">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>30</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="karta_igraca_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="karta_igraca_4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>310</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="karta_igraca_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="karta_igraca_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="karta_racunara_4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="karta_racunara_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="karta_racunara_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="karta_racunara_1">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="karta_racunara_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sakrivena_karta">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nova_igra">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Nova igra</string>
   </property>
  </widget>
  <widget class="QLabel" name="rezultat">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#eeeeec;&quot;&gt;Trenutno stanje:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="novac">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="karta_igraca_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="karta_racunara_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pobedio_racunar">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cip50">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cip100">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ulog50">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ulog100">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>420</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cip50_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>460</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cip100_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>460</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>60</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="racunar100">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>190</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="racunar50">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
